query-engine/connectors/mongodb-query-connector/src/interface/transaction.rs

```diff
@@ -39,17 +39,21 @@ impl<'conn> MongoDbTransaction<'conn> {
 
 #[async_trait]
 impl<'conn> Transaction for MongoDbTransaction<'conn> {
-    async fn commit(&mut self) -> connector_interface::Result<()> {
+    async fn begin(&mut self) -> connector_interface::Result<()> {
+        Ok(())
```

Comment:
Should it be an error instead?

Reply:
I'm not sure, what would that code look like?